src/app/services/node_selection_service.js

```diff
@@ -36,7 +36,7 @@ angular
         options: {
             packages: [],
             tags: [null],
-            resource_types: ['model', 'seed', 'snapshot', 'source', 'test', 'analysis', 'exposure', 'metric'],
+            resource_types: ['model', 'seed', 'snapshot', 'source', 'test', 'analysis', 'exposure', 'metric'].sort(),
```

Comment:
Let's take the sort off of this one.  This order is purposeful in the what users are generally looking for.  Most of the time it's a `model` and having it at the top prevents having top hunt down the word.

Reply:
Resolved: 5be3802bf59826ea0ae931c643990c16e50ad288.

I appreciate there is some logic here and that most of the time people (including me!) are looking for the "models" item however this is the only place I know of where the order is not alphabetical and that's a bit of an unexpected inconvenience.